library/src/main/java/com/schibsted/spain/barista/internal/matcher/TextColorMatcher.kt

```diff
@@ -15,7 +14,14 @@ class TextColorMatcher(@ColorRes private val color: Int) : BoundedMatcher<View,
     override fun matchesSafely(textView: TextView): Boolean {
         val resources = textView.context.resources
         colorName = resources.getResourceEntryName(color)
-        return textView.currentTextColor == ContextCompat.getColor(textView.context, color)
+        val colorMatches = textView.currentTextColor == ContextCompat.getColor(textView.context, color)
+        return if (colorMatches) {
+            true
+        } else {
+            val colorStateList = ContextCompat.getColorStateList(textView.context, color)
+            val colorForState = colorStateList.getColorForState(intArrayOf(-android.R.attr.enabled), colorStateList.defaultColor)
```

Comment:
Emmm what if the stat is not enabled? Or any other state? I guess this only works if the current color from the list is enabled or the default.